### PR TITLE
Add expires example to proof.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #313 by adding a proof to the specification that uses the `expires` property. 

/cc @filip26

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/314.html" title="Last updated on Oct 14, 2024, 9:32 PM UTC (00e0119)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/314/25e2237...00e0119.html" title="Last updated on Oct 14, 2024, 9:32 PM UTC (00e0119)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -926,6 +926,27 @@ <h3>Proofs</h3>
 representing time values to individuals.
         </p>
 
+        <pre class="example nohighlight"
+        title="A proof attached to a data document that uses the 'expires' property">
```
Comment (TallTed) [inline index.html]:
```suggestion
        title="A proof that uses the 'expires' property, attached to a data document">
```

Comment (TallTed) [reply index.html]: or
```suggestion
        title="A data document with an attached proof that uses the 'expires' property">
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Attempting to increase the clarity of the title

File: index.html [hunk 2]
```diff
@@ -926,6 +926,27 @@ <h3>Proofs</h3>
...
+  "@context": [
+    {"myWebsite": "https://vocabulary.example/myWebsite"},
+    "https://w3id.org/security/data-integrity/v2"
+  ],
+  "myWebsite": "https://hello.world.example/",
+  "proof": {
+    "type": "DataIntegrityProof",
+    "cryptosuite": "ecdsa-rdfc-2019",
+    "created": "2020-06-11T19:14:04Z",
+    <span class="comment">// the proof expires a month after it was issued</span>
```
Comment (dlongley) [inline index.html]:
```suggestion
    <span class="comment">// the proof expires a month after it was created</span>
```

File: index.html [hunk 3]
```diff
@@ -926,6 +926,27 @@ <h3>Proofs</h3>
...
+    <span class="highlight">expires": "2020-07-11T19:14:04Z"</span>,
```
Comment (dlongley) [inline index.html]:
```suggestion
    <span class="highlight">"expires": "2020-07-11T19:14:04Z"</span>,
```

**Review by dlongley — APPROVED** [review]

**Review by iherman — APPROVED** [review]

**Review by Wind4Greg — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**